### PR TITLE
add state dim changes - july

### PR DESCRIPTION
**Pull request by jimzers**
Commands for reproducing:

1. Shape generation
2. Pretraining
3. Training

1:
```
python main_DDPGfD.py --saving_dir variable_speed_07132021_v01 --hand_orientation normal --shapes CubeM --with_grasp_reward False --with_orientation_noise False --max_episode 100 --controller_type position-dependent --mode position-dependent
```

2:
```
python main_DDPGfD.py --saving_dir state_dim_variable_speed_07132021_v02 --hand_orientation normal --shapes CubeM --with_orientation_noise False --expert_replay_file_path "./experiments/position-dependent/no_noise/no_grasp/" --with_grasp_reward False --max_episode 100 --controller_type policy --mode pre-train --update_after 30 --eval_num 30 --state_range all_real
```

3:
```
python main_DDPGfD.py --saving_dir training_07132021_v02 --hand_orientation normal --shapes CubeM --with_orientation_noise True --expert_replay_file_path "./experiments/position-dependent/no_noise/no_grasp/" --agent_replay_buffer_path "./experiments/pre-train/state_dim_variable_speed_07132021_v02/replay_buffer/" --pretrain_policy_path "./experiments/pre-train/state_dim_variable_speed_07132021_v02/policy/pre-train_DDPGfD_kinovaGrip" --max_episode 100 --controller_type policy --update_after 6 --eval_num 5 --mode train --state_range all_real
```

### CODE REVIEW

**Review by nigelswenson — APPROVED** [review]

**Comment by hugheste** [comment]
> Commands for reproducing:
> 
> 1. Shape generation
> 2. Pretraining
> 3. Training
> 
> 1:
> 
> ```
> python main_DDPGfD.py --saving_dir variable_speed_07132021_v01 --hand_orientation normal --shapes CubeM --with_grasp_reward False --with_orientation_noise False --max_episode 100 --controller_type position-dependent --mode position-dependent
> ```
> 
> 2:
> 
> ```
> python main_DDPGfD.py --saving_dir state_dim_variable_speed_07132021_v02 --hand_orientation normal --shapes CubeM --with_orientation_noise False --expert_replay_file_path "./experiments/position-dependent/no_noise/no_grasp/" --with_grasp_reward False --max_episode 100 --controller_type policy --mode pre-train --update_after 30 --eval_num 30 --state_range all_real
> ```
> 
> 3:
> 
> ```
> python main_DDPGfD.py --saving_dir training_07132021_v02 --hand_orientation normal --shapes CubeM --with_orientation_noise True --expert_replay_file_path "./experiments/position-dependent/no_noise/no_grasp/" --agent_replay_buffer_path "./experiments/pre-train/state_dim_variable_speed_07132021_v02/replay_buffer/" --pretrain_policy_path "./experiments/pre-train/state_dim_variable_speed_07132021_v02/policy/pre-train_DDPGfD_kinovaGrip" --max_episode 100 --controller_type policy --update_after 6 --eval_num 5 --mode train --state_range all_real
> ```

Do you mean to have --update_after 30 --eval_num 30 in the Pre-training command and --update_after 6 --eval_num 5 in the Training command? These are different than the default of --update_after 100 and --eval_num 100. Also, the Pre-training command will need to have --expert_prob 0.

Otherwise, looks great! Nice work! :)

**Review by hugheste — APPROVED** [review]
